src/bin/client-demo.rs

```diff
@@ -93,9 +93,20 @@ fn main() {
     let last_id = acc.get_last_id().wait().unwrap();
     println!("Got last ID {:?}", last_id);
 
+    let rnd = GenKeys::new(demo.mint.keypair().public_key_bytes());
+    let tokens_per_user = 1_000;
+
+    let users: Vec<_> = (0..demo.num_accounts)
+        .into_par_iter()
```

Comment:
I see you use into_par_iter() here, but not genesis-demo? Somehow the same keys are generated?

Reply:
The same `n` keys are generated, I use a set in the test to establish this since they could theoretically provide 100 of the same keys in different sequences